app/views/businessactivities/expected_amls_turnover.scala.html

```diff
@@ -31,13 +31,21 @@
     heading = header
 ) {
 
-        <ul class="list list-bullet">
-            @bmBusinessActivities.map {activity =>
-              @activity.split('|').map {a =>
-                <li>@a</li>
-              }
+    @bmModel.activities.map { a =>
+        @if(a.businessActivities.size > 1) {
```

Comment:
Is a.businessActivities identical to businessTypes?

Reply:
No, businessTypes is linked to a method within the expected amls turnover controller which takes the activity and maps it to a message.

a.businessActivities links to a getMessage method within the businessActivities model. 

Both methods do look very similar in that they check for a businessType and => to a message specific to that type, they're just in different places so maybe this could be refactored. How though i'm unsure.